### DISCUSSION TITLE
@ProtoAdapter can only be applied to classes.

### DISCUSSION THREAD

**Discussion by minhtranes** [infinispan/infinispan#10547] [Q&A]
I encountered the issue when trying to use generate proto adapter for embedded cache on infinispan version `14.0.3.Final`.
For detail: the error happens when I use `@AutoProtoSchemaBuilder` with `basePackages` but `includeClasses`.

**Comment by tristantarrant** [comment]
If could provide a snippet of code that causes the issue together with the full exception, that would be great
